thoth-api/src/graphql/model.rs

```diff
@@ -390,6 +400,18 @@ impl QueryRoot {
         }
     }
 
+    #[graphql(description = "Query a single work using its DOI")]
+    fn work_by_doi(context: &Context, doi: String) -> FieldResult<Work> {
+        let connection = context.db.get().unwrap();
+        match crate::schema::work::dsl::work
+            .filter(crate::schema::work::dsl::doi.ilike(format!("%{}%", doi)))
```

Comment:
why doing a `LIKE` statement instead of `=` (`.eq()`)?

Reply:
Oops, I'll fix that.